composer.json

```diff
@@ -17,14 +17,14 @@
   "require": {
     "php": "^7.1|^8.0",
```

Comment:
https://github.com/guzzle/psr7/blob/master/CHANGELOG.md#200beta-1---2021-03-21

`guzzlehttp/psr7` drops support for PHP versions < 7.2.

Although it may still work on PHP 7.1 depending on whether the user does or doesn't rely on PHP 7.1 functionality, should we increase our PHP requirement to `"^7.2|^8.0"`